scripts/template_upload_all.py

```diff
@@ -58,6 +58,8 @@ def parse_cmd_line():
                         help='local yaml file path, to use local provider_data & not conf/cfme_data'
                              'to be useful for template upload/deploy by non cfmeqe',
                         default=None)
+    parser.add_argument('--print-name-only', dest='print_name_only', action="store_true",
```

Comment:
I've got outstanding branches for some major updates to the template name method, and the addition of this gives me even more reason to move that function out of this script. Not something to necessarily scope here since the existing template_name method is way too integrated into the script, just to be aware :)

I'm thinking it would fit in cfme/utils/trackerbot.py decently, since that's what these names are for, our recording in trackerbot.